lib/beaker/shared/options_resolver.rb

```diff
@@ -0,0 +1,21 @@
+module Beaker
+  module Shared
+    # Methods for parsing options.
+    # - determine if a mode should be run in parallel
```

Comment:
This particular comment seems out of place; I think this needs to be moved to the `run_in_parallel?` method, not `OptionsResolver` module.